parser-utils/Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "mentat_parser_utils"
 version = "0.1.0"
-authors = ["Victor Porof <vporof@mozilla.com>"]
+authors = ["Victor Porof <vporof@mozilla.com>", "Richard Newman <rnewman@mozilla.com>"]
```

Comment:
I'm assuming that these will be useful for people asking for review on changes. Is that a valid assumption?